packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -189,10 +188,8 @@ export class GutenbergEditorPage extends BaseContainer {
 	 * @returns {Promise<void>} No return value.
 	 */
 	async _visitPublishedEntryFromPublishPane(): Promise< void > {
-		await Promise.all( [
-			this.frame.click( selectors.viewPostButton ),
-			this.page.waitForNavigation(),
-			this.page.waitForLoadState( 'networkidle' ),
-		] );
+		const viewPostButton = await this.frame.waitForSelector( selectors.viewPostButton );
+		await Promise.all( [ this.page.waitForNavigation(), viewPostButton.click() ] );
```

Comment:
Let's also call the click [directly from the `frame`](https://playwright.dev/docs/api/class-frame#frame-click). The above two lines are an equivalent to:
```suggestion
		await Promise.all( [
			this.page.waitForNavigation(),
			this.frame.click( selectors.viewPostButton ),
		] );
```

Reply:
Done!